crates/project-model/src/tests.rs

```diff
@@ -46,6 +46,39 @@ fn load_cargo_with_overrides(
     to_crate_graph(project_workspace)
 }
 
+fn load_cargo_with_fake_sysroot(
```

Comment:
do we still need this function or can we just re-use `load_rust_project`

Reply:
I cleaned up this test file, the tests didn't need the fake sysroot, but I had to split up and use `load_cargo_with_overrides` to make it work.

Reply:
oops, it seems i broke something